CPAC/pipeline/utils.py

```diff
@@ -20,10 +20,41 @@
 
 from CPAC.func_preproc.func_motion import motion_estimate_filter
 from CPAC.utils.bids_utils import insert_entity
+from nipype import Node, Workflow, Function
+from CPAC.pipeline import nipype_pipeline_engine as pe
+from nipype.interfaces import afni
+from nipype.interfaces.afni import Info
+import os
+import pandas as pd
 
 MOVEMENT_FILTER_KEYS = motion_estimate_filter.outputs
 
 
+def find_orientation(input_file):
+    import subprocess
+    cmd_3dinfo = [
+        "3dinfo",
+        "-orient", input_file
+    ]
+
+    orientation = subprocess.run(cmd_3dinfo, capture_output=True, text=True).stdout.strip().upper()
+    return orientation
+    
+def check_all_orientations(input_images:list, desired_orientation:str="RPI"):
+    desired_orientation = desired_orientation.upper()
+    orientations = []
+    find_orient = Node(Function(input_names=["input_file"],
+                            output_names=["orientation"],
+                            function=find_orientation),
+                    name="find_orient")
```

Comment:
```suggestion
    find_orient = Node(
        Function(
            input_names=["input_file"],
            output_names=["orientation"],
            function=find_orientation,
        ),
        name="find_orient",
    )
```

[pre-commit](https://github.com/FCP-INDI/C-PAC/blob/dbdacd1c6a745b4847b2c6abdf2c2f3d6150511f/CONTRIBUTING.md#pre-commit) should have caught this?